### PR TITLE
Use hexadecimal format for opcode values

### LINKED ISSUE #82: Write opcode numbers in hexadecimal format
[Here](https://github.com/parallaxsecond/parsec-interface-rs/blob/2cb516f428d0057d11cb23a8059c72d6c17cbf9f/src/requests/mod.rs#L75) they are all written in base 10. To avoid mistakes like [this](https://github.com/parallaxsecond/parsec-book/pull/66#discussion_r492632320) it would be better to have them in hex.

### PR DESCRIPTION
**Pull request by hug-dev**
Also adds a link checker on the CI.

Close #82 

Signed-off-by: Hugues de Valon <hugues.devalon@arm.com>

### CODE REVIEW

File: src/requests/mod.rs [hunk 1]
```diff
@@ -74,45 +74,45 @@ pub enum BodyType {
...
+    PsaAeadEncrypt = 0x0011,
     /// PsaAeadDecrypt
-    PsaAeadDecrypt = 18,
+    PsaAeadDecrypt = 0x0012,
     /// PsaRawKeyAgreement operation
-    PsaRawKeyAgreement = 19,
+    PsaRawKeyAgreement = 0x0013,
     /// ListKeys operations
-    ListKeys = 26,
+    ListKeys = 0x001A,
```
Comment (paulhowardarm) [inline src/requests/mod.rs:115]: Any reason why there's a sudden discontiguous jump from 19 to 26 here? Are we anticipating some missing PSA Crypto operations in the gap?

Comment (ionut-arm) [reply src/requests/mod.rs:115]: @joechrisellis ^^^

Comment (ionut-arm) [reply src/requests/mod.rs:115]: It does seem as if some of the operations that are in the book aren't added here yet: https://parallaxsecond.github.io/parsec-book/parsec_client/operations/psa_verify_message.html this is 0x19

**Review by ionut-arm — APPROVED** [review]

File: src/requests/mod.rs (hunk 1 above)
Comment (samwell61) [reply src/requests/mod.rs:115]: The book contains documentation for all operations (and there are protobuf contracts too), including ones not yet present in Parsec but I had to pick op codes, which is why there is a gap. [e.g.](https://github.com/parallaxsecond/parsec-book/pull/61)

Comment (paulhowardarm) [reply src/requests/mod.rs:115]: Thank you very much! This looks fine to me, in that case.

**Review by paulhowardarm — APPROVED** [review]

**Pull request merged**